kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -124,6 +124,7 @@ abstract class AbstractOperation(session: Session) extends Operation with Loggin
 
   protected def setOperationException(opEx: KyuubiSQLException): Unit = {
     this.operationException = opEx
+    withOperationLog(error(s"Error operating $opType: ${opEx.getMessage}", opEx))
```

Comment:
Due I do not find the batch operation exception from kyuubi log(I got it from kyuubi operation event stored in elasticsearch), so I think we can log the ex here.